go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/go-logr/logr
 
-go 1.14
+go 1.16
```

Comment:
@thockin: I know considered this a "trivial" change, but the reason for it is not clear to me. Why was it done, and why 1.16?

The "go" directive in a go.mod file does not specify a minimum compiler version, but rather the version of the Go language as used in the project: https://github.com/golang/go/issues/30791

I'm not sure what that means for functions only added in later releases. Those are not language features, are they?